nexus/proposals/api.py

```diff
@@ -17,11 +17,16 @@ class ProposalViewSet(mixins.ListModelMixin,
                       mixins.DestroyModelMixin,
                       viewsets.GenericViewSet):
     queryset = models.Proposal.objects.all()
-    serializer_class = serializers.ProposalSerializer
+
+    def get_serializer_class(self):
+        if self.action in ['accept', 'retract']:
+            return serializers.StatusUpdateSerializer
+        else:
+            return serializers.CreateSerializer
```

Comment:
What is `CreateSerializer`?

Reply:
I replaced `ProposalSerializer` with 2 serializers. One that comes in use when a proposal is created.
https://github.com/CuriousLearner/nexus/pull/11/files#diff-1b42a777daae5eec2c8eb789a4b1f06eR9

Reply:
I replaced `ProposalSerializer` with 2 serializers. One that comes in use when a proposal is created.
https://github.com/CuriousLearner/nexus/pull/11/files#diff-1b42a777daae5eec2c8eb789a4b1f06eR9

Reply:
Name them correctly. `CreateSerializer` doesn't mean anything.

Reply:
Fixed :)